src/styles/utilities/_extends.scss

```diff
@@ -0,0 +1,23 @@
+%size-description-code {
+  padding-top: 4px;
+  font-size: 10px;
+  line-height: 14px;
+  color: $secondary-color;
+}
+
+%size-description-title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+%size-price-cost {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+%size-price-name {
```

Comment:
The name of the placeholder is not descriptive enough. Consider renaming it to something more descriptive like '%price-name-style'.